app/controllers/application_controller.rb

```diff
@@ -3,9 +3,17 @@ class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
   helper_method :current_person
 
+  rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
```

Comment:
Originally I had set up the custom error page per:
https://thepugautomatic.com/2014/08/404-with-rails-4/ 
and updated the application config file to include ```config.action_dispatch.rescue_responses["Pundit::NotAuthorizedError"] = :forbidden``` per:
https://github.com/elabs/pundit#rescuing-a-denied-authorization-in-rails

This allowed us to keep the default static pages as well as use a custom dynamic error page.

However when the error was raised there was no redirect to the 403 route, and the template was rendered on the same page where the error was raised. 

Using this method, gave a redirect, and removed some of the complexity  in setting up the custom dynamic error page.  It does however add complexity to the application controller and if we also wanted to add other custom error pages, it may need to be changed back to the above approach anyway.

I will add another commit to use the original approach so you can compare them side-by-side and we can discuss the pros/cons of each approach.